src/hooks/screen/useChatScreenHook.ts

```diff
@@ -146,7 +183,13 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
     goToChatInfoScreen,
     sendChat,
     handleUserName,
-    updateChatContinuity
+    updateChatContinuity,
+    flatListRef,
+    scrollContext: {
+      selectedMessageId,
+      setSelectedMessageId,
+      handleScrollTo
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The return object of the hook is updated to include a reference to the FlatList component and a context for scrolling. This seems like a useful addition for managing scroll behavior, but make sure that these new properties are used appropriately in the rest of the application.
```diff
+     updateChatContinuity,
+     flatListRef,
+     scrollContext: {
+       selectedMessageId,
+       setSelectedMessageId,
+       handleScrollTo
+     }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->